client/app/bundles/course/assessment/submission/propTypes.js

```diff
@@ -6,6 +6,29 @@ const optionShape = PropTypes.shape({
   correct: PropTypes.bool,
 });
 
+const logShape = PropTypes.shape({
```

Comment:
All these needed?

Reply:
I was thinking to make it more modular, and also since if you look into the file `LogsIndex.tsx`, I only pass the information contained in here (main information) into `LogsHead.tsx` and only pass the information regarding all the logs into `LogsContent.tsx`.

Reply:
What do you mean by more modular? this proptypes is used for javascript file. We are already using typescript and the types have been declared in `client/app/types/course/assessment/submission/logs.ts`

Reply:
oh I see. I'm sorry I thought this will be visited as well (my bad I now understand completely about this). Removed this already